packages/uploadthing/src/_internal/deprecations.ts

```diff
@@ -0,0 +1,4 @@
+export const logDeprecationWarning = (message: string) => {
+  // eslint-disable-next-line no-console
+  console.warn(`⚠️ [uploadthing] ${message}`);
```

Comment:
given that this ie explicitly called "logDeprecationWarning" should we also add a tag [deprecated]  or similar?